tests/RegressionTests_hera.intel.log

```diff
@@ -2095,21 +2095,21 @@ Checking test 036 fv3_wrtGauss_netcdf results ....
  Comparing RESTART/sfc_data.tile5.nc .........OK
  Comparing RESTART/sfc_data.tile6.nc .........OK
 
-  0: The total amount of wall time                        = 130.582460
+  0: The total amount of wall time                        = 38.047905
```

Comment:
I wonder how the timing change is so big.

Reply:
I have seen considerably variability in tests/runs that are I/O heavy on hera recently, maybe that is the reason?

Reply:
Yeah, it might be. The running time on dell is pretty stable.

Reply:
The files in `develop-20210513/INTEL/fv3_wrtGauss_netcdf` are actually identical to the files in `develop-20210512/INTEL/fv3_wrtGauss_netcdf`. So no change in results for this test.